### PR TITLE
language.h: update documentation / URL für K8200

### PR DESCRIPTION
**Pull request by CONSULitAS**
language.h: update documentation for LANGUAGE_INCLUDE (@anhardt correct?)
language.h: URL für K8200 github repository

### CODE REVIEW

File: Marlin/language.h [hunk 1]
```diff
@@ -32,7 +32,7 @@
 // kana_utf Japanese
 
 #ifndef LANGUAGE_INCLUDE
-  // pick your language from the list above
+  // pick your default from the list above, individual language ist set in LCD section of Configuration.h
```
Comment (thinkyhead) [inline Marlin/language.h]: Neither comment is really needed. This is not meant to be edited.

File: Marlin/language.h [hunk 2]
```diff
@@ -31,8 +31,8 @@
 // kana     Japanese
 // kana_utf Japanese
 
+// fallback if no language is set, don't change
```
Comment (CONSULitAS) [inline Marlin/language.h:34]: @thinkyhead What do you think? I think there should be at least a warning like
this.

Comment (KiteLab) [inline Marlin/language.h:34]: @CONSULitAS 
Even if changed there https://github.com/CONSULitAS/Marlin-K8200/blob/Development_Update_language.h/Marlin/language.h#L238 will work.

Comment (thinkyhead) [inline Marlin/language.h:34]: That's good. I like documentation comments more every day.

**Pull request merged**